youtube_dl/extractor/francetv.py

```diff
@@ -360,3 +360,73 @@ def _real_extract(self, url):
             webpage, 'video id').split('@')
 
         return self._extract_video(video_id, catalogue)
+
+
+class JeunesseBaseIE(FranceTVBaseInfoExtractor):
+    def _real_extract(self, url):
+        id = self._match_id(url)
+        entries = []
+
+        playlist = self._download_json(url.split(id)[0] + id + '/playlist', id)
+
+        if not playlist.get('count'):
+            raise ExtractorError('%s is not available' % id, expected=True)
+
+        for item in playlist.get('items'):
+            video_id, catalogue = item['identity'].split('@')
+            try:
+                entries.append(self._extract_video(video_id, catalogue))
+            except Exception:
```

Comment:
Already pointed out: must be `ExtractorError`.

Reply:
and I commented about that, any suggestion in order to not break the download ?